### PR TITLE
fix linter errors

### PR DESCRIPTION
**Pull request by m00g3n**
Fix linter errors pahse1

Changes proposed in this pull request:

- add required `apiVersion` elements
- regenerate outdated jsonschema
- remove `Release.Time.Seconds` to have `helm3` chart compatibility

**Related issue(s)**
See also : https://github.com/kyma-project/kyma/issues/7933

### CODE REVIEW

File: resources/cluster-essentials/charts/pod-preset/Chart.yaml [hunk 1]
```diff
@@ -1,4 +1,6 @@
 name: pod-preset
-description: A Helm chart for PodPreset resource that's implemented via a CRD that is paired with a mutating webhook in order to modify pod specs on the fly
+description: A Helm chart for PodPreset resource that's implemented via a CRD that
```
Comment (Demonsthere) [inline resources/cluster-essentials/charts/pod-preset/Chart.yaml:2]: For a multiline string shouldn't this look like

```yaml
description: |
  foo
```

Comment (m00g3n) [reply resources/cluster-essentials/charts/pod-preset/Chart.yaml:2]: thx it looks like autoformatter

**Pull request merged**